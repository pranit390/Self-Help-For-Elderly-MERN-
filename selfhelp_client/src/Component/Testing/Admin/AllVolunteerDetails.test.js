import React from 'react';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
Enzyme.configure({ adapter: new Adapter() });
import {shallow} from 'enzyme'

import AllVolunteersDetails from '../../Admin/AllVolunteersDetails.js';

describe("All volunteer details", () => {
    it("should render my component", () => {
      const wrapper = shallow(<AllVolunteersDetails/>);
      expect(wrapper.getElements()).toMatchSnapshot();
    });
  });