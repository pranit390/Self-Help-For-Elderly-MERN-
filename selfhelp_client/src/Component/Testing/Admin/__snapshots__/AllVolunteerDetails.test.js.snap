// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`All volunteer details should render my component 1`] = `
Array [
  <styled.div>
    <React.Fragment>
      <AdminNavigationBar />
      <br />
      <br />
      <br />
      <br />
      <br />
      <br />
      <Container
        fluid={false}
      >
        <h3
          class="h3"
        >
          Volunteer Details
        </h3>
        <table
          className="table table-striped"
          style={
            Object {
              "marginTop": 20,
            }
          }
        >
          <thead>
            <tr>
              <th>
                First Name
              </th>
              <th>
                Last Name
              </th>
              <th>
                Gender
              </th>
              <th>
                Phone number
              </th>
              <th>
                Email
              </th>
              <th>
                Profession
              </th>
              <th>
                Action
              </th>
            </tr>
          </thead>
          <tbody />
        </table>
        <o
          activeClassName="active"
          breakClassName="break-me"
          breakLabel="..."
          containerClassName="pagination"
          disableInitialCallback={false}
          disabledClassName="disabled"
          eventListener="onClick"
          marginPagesDisplayed={2}
          nextAriaLabel="Next page"
          nextClassName="next"
          nextLabel="next"
          nextRel="next"
          onPageChange={[Function]}
          pageCount={10}
          pageLabelBuilder={[Function]}
          pageRangeDisplayed={5}
          prevRel="prev"
          previousAriaLabel="Previous page"
          previousClassName="previous"
          previousLabel="prev"
          subContainerClassName="pages pagination"
        />
      </Container>
      <Footer />
    </React.Fragment>
  </styled.div>,
]
`;
